src/components/soshistorypage/HistoyFilter.tsx

```
@@ -0,0 +1,135 @@
+import { useState } from 'react';
+import classNames from 'classnames';
```

ESLint

[error] `classnames` import should occur before import of `react`

(import/order)